definitions/private-law/json/AuthorisationCaseField/DraftAnOrder/AuthorisationCaseField.json

```diff
@@ -89,5 +89,41 @@
     "CaseFieldID": "draftOrderCollection",
     "UserRole": "caseworker-privatelaw-judge",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "10/08/2021",
+    "CaseTypeID": "PRLAPPS",
+    "CaseFieldID": "c43Label",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "caseworker-privatelaw-systemupdate",
+          "caseworker-wa-task-configuration",
+          "[SOLICITORA]",
+          "[SOLICITORB]",
+          "[SOLICITORC]",
+          "[SOLICITORD]",
+          "[SOLICITORE]",
+          "[SOLICITORF]",
+          "[SOLICITORG]",
+          "[SOLICITORH]",
+          "[SOLICITORI]",
+          "[SOLICITORJ]",
+          "[APPLICANTSOLICITOR]",
+          "[CREATOR]",
+          "caseworker-privatelaw-judge",
+          "caseworker-privatelaw-courtadmin"
```

Comment:
LA should have access as well, PLease make sure to add LA access to fields/events for which judge has got access. As LA/Judge will have same level of access as they fall under gatekeeper category